From e4aa6899c5bea2e9acd3a821c8d3d7803ac844cb Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Mon, 12 Dec 2022 11:46:20 +0200
Subject: [PATCH] [SW-102640] gaudi3: increase stack size

The current stack size (2K) is not enough for
function boot_info_finalize_no_merge(), which requires
about 3.6K

Change-Id: I41ad3ad9b462ed88f38b242a1c3c2a97b1c261a6
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/arch/gaudi3/arc/ppboot.lds.S b/arch/gaudi3/arc/ppboot.lds.S
index 09d4077..774d6b9 100644
--- a/arch/gaudi3/arc/ppboot.lds.S
+++ b/arch/gaudi3/arc/ppboot.lds.S
@@ -13,7 +13,7 @@
 MEMORY
 {
 	.sram 	: ORIGIN = PPBOOT_TEXT_BASE, LENGTH = PPBOOT_SIZE
-	DCCM	: ORIGIN = PPBOOT_DCCM_BASE, LENGTH = 8K
+	DCCM	: ORIGIN = PPBOOT_DCCM_BASE, LENGTH = 10K
 }
 
 REGION_ALIAS("REGION_DATA",	DCCM)
@@ -63,7 +63,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 2048 ;
+		. = . + 4096 ;
 		__stack_top = . ;
 	} > REGION_STACK
 
